### PR TITLE
feat: add Producer

### PR DESCRIPTION
**Pull request by pgallik**

### CODE REVIEW

File: src/StreetNameRegistry.Producer/StreetNameRegistry.Producer.csproj [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <Compile Include="..\..\SolutionInfo.cs" Link="Properties\SolutionInfo.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\StreetNameRegistry.Infrastructure\StreetNameRegistry.Infrastructure.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Migrations\" />
+  </ItemGroup>
```
Comment (ArneD) [inline src/StreetNameRegistry.Producer/StreetNameRegistry.Producer.csproj:37]: Not needed

**Comment by lvermeulen** [comment]
lgtm, na resolve

File: src/StreetNameRegistry.Producer/StreetNameRegistry.Producer.csproj [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Import Project="..\..\packages\Be.Vlaanderen.Basisregisters.Build.Pipeline\Content\Be.Vlaanderen.Basisregisters.Build.Pipeline.Settings.App.props" />
+
+  <PropertyGroup>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
+    <EnableDefaultContentItems>false</EnableDefaultContentItems>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>bin\Debug\net5.0\MunicipalityRegistry.Producer.xml</DocumentationFile>
```
Comment (ArneD) [inline src/StreetNameRegistry.Producer/StreetNameRegistry.Producer.csproj:10]: `StreetNameRegistry`

File: src/StreetNameRegistry.Producer/StreetNameRegistry.Producer.csproj [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <NoWarn>1701;1702;1705;1591</NoWarn>
+    <DefineConstants>TRACE;DEBUG;NETCOREAPP;NET5_0</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <DocumentationFile>bin\Release\net5.0\MunicipalityRegistry.Producer.xml</DocumentationFile>
```
Comment (ArneD) [inline src/StreetNameRegistry.Producer/StreetNameRegistry.Producer.csproj:15]: `StreetNameRegistry`

File: src/StreetNameRegistry.Producer/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
+{
+  "ConnectionStrings": {
+    "Events": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.MunicipalityRegistry;Trusted_Connection=True;",
+    "ProducerProjections": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.MunicipalityRegistry;Trusted_Connection=True;",
+    "ProducerProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.MunicipalityRegistry;Trusted_Connection=True;"
+  },
```
Comment (ArneD) [inline src/StreetNameRegistry.Producer/appsettings.json:6]: `StreetNameRegistry`

**Pull request merged**